staging/src/k8s.io/apiserver/pkg/util/flag/langle_separated_map_string_string.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package flag
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+)
+
+// LangleSeparatedMapStringString can be set from the CLI with "<" instead of "="
+// as the key-value separator. Pairs are still separated with ",".
+type LangleSeparatedMapStringString map[string]string
+
+// String implements github.com/spf13/pflag.Value
+func (m LangleSeparatedMapStringString) String() string {
+	pairs := []string{}
+	for k, v := range m {
+		pairs = append(pairs, fmt.Sprintf("%s<%s", k, v))
+	}
+	sort.Strings(pairs)
+	return strings.Join(pairs, ",")
+}
+
+// Set implements github.com/spf13/pflag.Value
+func (m LangleSeparatedMapStringString) Set(value string) error {
+	// clear old values
+	for k := range m {
```

Comment:
Should only be done on first call to Set

Reply:
Hmm, yeah I guess it's useful to allow multiple invocations for these too. Will do.

Reply:
Done